etl/firefox_legacy_etl.py

```diff
@@ -69,7 +69,12 @@ def write_firefox_legacy_metadata(output_dir, functions_dir):
         with open(os.path.join(probe_output_directory, f"data_{probe_name}.json"), "w") as f:
             json.dump(probe_metadata, f)
 
-    # write a search index for the app
+    # write a search index for legacy telemetry data
```

Comment:
Why do we need to need keep the search that only includes legacy telemetry data?

Reply:
We're creating the legacy + fog API for GLAM only because of its specific use case. I wanted to still keep the APIs for individual products for other use cases as that was the original intention behind the Glean Dictionary search service: to let users search through metrics for each product.

Reply:
> that was the original intention behind the Glean Dictionary search service: to let users search through metrics for each product.

Is this documented somewhere? If not, please file a follow-up bug to do that.

Reply:
It's documented [here](https://github.com/mozilla/glean-dictionary#search-service), albeit not very thorough.